src/m3u8/parse-stream.js

```diff
@@ -280,7 +280,9 @@ export default class ParseStream extends Stream {
         event.attributes = parseAttributes(match[1]);
         // parse the IV string into a Uint32Array
         if (event.attributes.IV) {
-          if (event.attributes.IV.substring(0, 2) === '0x') {
+          let prefix = event.attributes.IV.substring(0, 2);
+
+          if (prefix === '0x' || prefix === '0X') {
```

Comment:
You should do `prefix = event.attributes.IV.substring(0, 2).toLowerCase()` and simplify the condition here to `prefix === '0x'`.